src/rendergraph/opengl/rendergraph/openglnode.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "backend/baseopenglnode.h"
+
+namespace rendergraph {
+using OpenGLNode = BaseOpenGLNode;
+} // namespace rendergraph
```

Comment:
There are many files with less than 10 lines. Are you going to extend these files in the subsequent PRs? Because this makes the code harder to follows and reeks of YAGNI.

Reply:
This is a consequence of the design of having two backend implementations, and a thin common API to interface with either. I don’t see away around this with the current design.